Tasks/AzureAppConfigurationSnapshotV1/taskController.ts

```diff
@@ -0,0 +1,65 @@
+import * as tl from "azure-pipelines-task-lib/task";
+import { AppConfigurationClient, CreateSnapshotResponse } from '@azure/app-configuration';
+import { RestError } from "@azure/core-rest-pipeline";
+import { TaskParameters } from './taskParameters';
+import { AppConfigurationError, getErrorMessage } from "./errors";
+import { Utils } from './utils';
+
+export class TaskController {
+    private _taskParameters: TaskParameters;
+    private _client: AppConfigurationClient;
+
+    constructor(taskParameters: TaskParameters) {
+        this._taskParameters = taskParameters;
+        this._client = new AppConfigurationClient(
+            taskParameters.configStoreUrl,
+            taskParameters.credential,
+            {
+                userAgentOptions: {
+                    userAgentPrefix: Utils.GenerateUserAgent()
+                }
+            }
+        );
+    } 
+
+    public async createSnapshot(): Promise<void> {
+        console.log(tl.loc("SnapshotTaskIsStartingUp"));
```

Comment:
Can we use sentence structure for the logs where it can apply? `The snapshot task is starting up."